Header/Header.js

```diff
@@ -0,0 +1,143 @@
+import Slottable from '@enact/ui/Slottable';
+
+import Skinnable from '../Skinnable';
+
+import componentCss from './Header.module.less';
+
+/**
+ * A header component for a Panel with a `title`, `titleAbove`, `subtitle`
+ *
+ * @class Header
+ * @memberof agate/Header
+ * @ui
+ * @public
+ */
+const HeaderBase = kind({
+	name: 'Header',
+
+	propTypes: /** @lends agate/Header.Header.prototype */ {
+		/**
+		 * Title of the header.
+		 *
+		 * This is a [`slot`]{@link ui/Slottable.Slottable}, so it can be used as a tag-name inside
+		 * this component.
+		 *
+		 * Example:
+		 * ```
+		 *  <Header>
+		 *  	<titleAbove>Introducing...</titleAbove>
+		 *  	<title>Example Header Title</title>
+		 *  	<subtitle>The Adventure Continues</subtitle>
+		 *  </Header>
+		 * ```
+		 *
+		 * @type {String}
+		 */
+		title: PropTypes.string.isRequired,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal Elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `header` - The root class name
+		 * * `titleContainer` - The background title container
+		 * * `subtitle` - Applied to `subtitle`
+		 * * `title` - Applied to `title`
+		 * * `titleAbove` - Applied to `titleAbove`
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		css: PropTypes.object,
+
+		/**
+		 * Hides the horizontal-rule (line) under the component
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		hideLine: PropTypes.bool,
+
+		/**
+		 * Text displayed below the title.
+		 *
+		 * This is a [`slot`]{@link ui/Slottable.Slottable}, so it can be used as a tag-name inside
+		 * this component.
+		 *
+		 * @type {String}
+		 */
+		subtitle: PropTypes.string,
+
+		/**
+		 * Text displayed above the title.
+		 *
+		 * This is a [`slot`]{@link ui/Slottable.Slottable}, so it can be used as a tag-name inside
+		 * this component.
+		 *
+		 * @type {String}
+		 */
+		titleAbove: PropTypes.string
+	},
+
+	styles: {
+		css: componentCss,
+		className: 'header',
+		publicClassNames: true
+	},
+
+	computed: {
+		className: ({hideLine, styler}) => styler.append({hideLine, standard: true}),
+		subtitleComponent: ({css, subtitle}) => {
+			return (subtitle != null && subtitle !== '') ? <h2 className={css.subtitle}>{subtitle}</h2> : null;
+		},
+		titleAboveComponent: ({css, titleAbove}) => {
+			return (titleAbove != null && titleAbove !== '') ? <h2 className={css.titleAbove}>{titleAbove}</h2> : null;
+		}
+	},
+
+	render: ({children, css, title, titleAboveComponent, subtitleComponent, ...rest}) => {
+		delete rest.hideLine;
+		delete rest.subtitle;
+		delete rest.titleAbove;
+
+		return (
+			<Row component="header" aria-label={title} {...rest}>
+				<Column className={css.titleContainer}>
+					{titleAboveComponent}
+					<h1 className={css.title}>{title}</h1>
+					{subtitleComponent}
+				</Column>
+				{children ? <Layout className={css.endSlot} shrink>{children}</Layout> : null}
```

Comment:
The usage of `shrink` here is causing a prop bleed error when attempting to render `children`, since `shrink` is a property of `Cell`, not `Layout`. We may be able to just remove the prop altogether, unless there was a reason for the initial use of it. I think things would be ok without it.

Reply:
Yea that was cruft when it was a `Cell`. I can remove the prop.